PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2354,7 +2354,7 @@ class PurchasesTests: XCTestCase {
         setupPurchases()
 
         for key in data.keys {
-            expect(self.backend.invokedPostAttributionDataParameters?.data?.keys.contains(key)) == true
+            expect(self.backend.invokedPostAttributionDataParameters?.data?.keys.contains(key)).toEventually(beTrue())
```

Comment:
we don't really need this `toEventually` to happen on all of them, right? 
perhaps we can instead have this once before the `for` loop
```swift
expect(self.backend.invokedPostAttributionData).toEventually(beTrue())
```
Which also indicates to readers that we're waiting for the method to have been invoked, before doing all of the checks. 
This also matches our expectation that once once parameter was set, they all have, we're not waiting until the next one

Reply:
Yeah, I was going to say it doesn't matter, BUT I really like your point about indicating to the reader that we're waiting for the method to be invoked and expect the rest to be there at that point. Updated!

Reply:
✅